client/blocks/reader-post-card/photo.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { noop } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * Internal Dependencies
+ */
+import cssSafeUrl from 'lib/css-safe-url';
+
+class PostPhoto extends React.Component {
+
+	constructor( props ) {
```

Comment:
FFTI but a cool little tidbit is that you could remove the constructor and just write:
```javascript
state = {
...
}````